config/routes.rb

```diff
@@ -0,0 +1,12 @@
+Rails.application.routes.draw do
+  root 'splash#show'
+  devise_for :users
+  resources :users
+  resources :tran_groups do
```

Comment:
- [ ] Let's kindly set up only necessary routes for the tran_groups resource as requested in the requirements. There's unnecessary space allocated to the unused routes which consumes up our memory.

Reply:
resolved